rcl_interfaces/msg/IntegerRange.msg

```diff
@@ -1,13 +1,22 @@
 # Represents bounds and a step value for an integer typed parameter.
 
-# Start value for valid values range, inclusive.
+# Start value for valid values, inclusive.
 int64 from_value
 
-# End value for valid values range, inclusive.
+# End value for valid values, inclusive.
 int64 to_value
 
 # Size of valid steps between the from and to bound.
-# Step should be positive, and a step of zero implies continuous values.
-# Step size should be less than the distance between the values.
-# The to_value is valid even if the step is not a multiple of it.
+# A step value of zero implies a continuous range of values.
+# Ideally, the step would be both less than or equal to the distance between the
```

Comment:
I know this is based on my suggestion originally, but now that I read it again, I think we should drop the word `both`, because `as well as` is sufficient and `both less than or equal` is a bit confusing on the first read.